### ISSUE TITLE
Incorrect namespace color for static method calls

### ISSUE THREAD

**Issue by philraj** [atom/language-php#284]
### Description

Namespaces are being improperly colored when being used in front of a static method call.

![screen shot 2017-10-04 at 6 03 26 pm](https://user-images.githubusercontent.com/16470658/31201891-62b0a072-a92e-11e7-9eb7-d80284986f9c.png)

```php
if($obj instanceof \One\Two\Three) {
  $val = \One\Two\Three::getVal();
}
```

**Comment by dsifford** [comment]
@philraj Just checked this one out again and it looks like it's actually correct in Atom, but broken in VSCode.

In your case, it'll likely be a theme issue again.

In my case, it's the result of VSCode being a couple commits behind master.

**Comment by philraj** [comment]
@dsifford You sure? When I inspect the namespace tokens I see the following classes only:
`syntax--constant syntax--other syntax--php`
Seems like they're being marked as constants instead of parts of a namespace.

**Comment by dsifford** [comment]
Ohhhhhhhhh, ok I didn't know you were talking about the namespace parts. I thought you were talking about the class that the static method was being called from.

Nevermind. Yeah, the namespace part is broken. You're right.

**Comment by dsifford** [comment]
Another broken think I'm seeing that might be related is simply calling static methods off of classes without leading namespaces. There's zero scope attached to the class in that situation...

```php
<?php
use Foo;
  Foo::bar();
//^^^broken
```

**Comment by 50Wliu** [comment]
VSCode uses their own TextMate-based engine, which differs from Atom's implementation.  Ingramz and I recently fixed a bug in the Atom implementation that was causing static methods to tokenize correctly in Atom but not in other TextMate-based engines.

I can confirm this bug and it looks like it'll be fixed by #281.

(Also, it's always super helpful if you could provide a code example that I can copy/paste!  Pictures help too, but I find examples to be the most useful.)

**Comment by philraj** [comment]
@50Wliu Of course, next time! :)
